pkg/kubectl/cmd/describe.go

```diff
@@ -64,6 +75,24 @@ func RunDescribe(f *cmdutil.Factory, out io.Writer, cmd *cobra.Command, args []s
 		return err
 	}
 
+	if len(args) == 0 {
+		fmt.Fprint(out, `
```

Comment:
I suppose fine to print this to stdout.

Most of this message is identical to the one for get. To avoid skew between them, can you store the common part (`Valid resource types include: ...`) in a constant and reuse it?